CONTRIBUTING.md

```diff
@@ -180,6 +181,15 @@ base](https://github.com/mlflow/mlflow/blob/cdc6a651d5af0f29bd448d2c87a198cf5d32
 For more information about Plugins, see
 <https://mlflow.org/docs/latest/plugins.html>.
 
+### Python Style Guide
+We follow [Google's python style guide](https://google.github.io/styleguide/pyguide.html) 
```

Comment:
We actually don't. We just use black, ruff, and pylint.

Reply:
Ok then we should list out our custom docstring style guide. Do we have this documented anywhere?

Reply:
No, let's document it :)

Reply:
Reworded to be more clear. Thanks for the comments!